modules/storage/mysql-single-server/README.md

```diff
@@ -0,0 +1,174 @@
+This module requires the namespace 'Microsoft.DBforMySQL' to be registered  for your subscription manually before deployment.
+
+## Parameters
+
+| Name                         | Type           | Required | Description                                                                                                                                                                                                                                                                                                                                                                                  |
+| :--------------------------- | :------------: | :------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `location`                   | `string`       | No       | Deployment region name. Default is the location of the resource group.                                                                                                                                                                                                                                                                                                                       |
+| `tags`                       | `object`       | No       | Deployment tags. Default is empty map.                                                                                                                                                                                                                                                                                                                                                       |
+| `administratorLogin`         | `string`       | Yes      | Required. The administrator username of the server. Can only be specified when the server is being created.                                                                                                                                                                                                                                                                                  |
+| `administratorLoginPassword` | `securestring` | Yes      | Required. The administrator password of the server. Can only be specified when the server is being created.                                                                                                                                                                                                                                                                                  |
```

Comment:
we prefer to minimize the required parameters.... I'm going to peek at other examples that we have to see what we do here. Default values for these would be great, but I can understand these 2 not having them... (the admin login password would be a guid generated from utc-now or something very random like that...)